test/json-ld.net.tests/json-ld.net.tests.csproj

```diff
@@ -24,6 +24,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+    <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
```

Comment:
New dependency to enable a decent UX in the case that developer doesn't have docker installed.

It has really good fundamentals as a random new dependency, though:
- NuGet package has 2.2 million downloads
- Project source is small, simple, and easy to understand
- Project was started (and is still maintained) by a developer on Visual Studio team
- Project has been active (and responsive to issues) over many years, as recently as 3 months ago